app/components/AnalystDashboard/AssessmentLead.tsx

```diff
@@ -50,8 +47,37 @@ const AssignLead: React.FC<Props> = ({
 
   const analystList = Object.keys(allAnalysts);
 
+  let backgroundColor: string;
```

Comment:
just wondering if we can use the `assessmentPillStyles.ts`?, since they seems to be almost the same and we get to control colors from a single source 🤔

Reply:
Yup, I'll refer to the file directly. Great catch